torch/distributed/_composable/fsdp/_fsdp_param_group.py

```diff
@@ -177,14 +189,24 @@ def pre_forward(
     def post_forward(self, module: nn.Module, input: Any, output: Any):
         with torch.profiler.record_function("FSDP::post_forward"):
             self.reshard()
+            self._record_post_forward()
             self._training_state = TrainingState.IDLE
             return output
 
+    def _record_post_forward(self) -> None:
```

Comment:
seems we are merging ``_ExecOrderData`` into ``FSDPParamGroup``. where would we place ``_backward_prefetch_limit`` / ``_forward_prefetch_limit``? I remember you mentioned rate limiter will be cuda stream based instead of cpu based?